src/lib.rs

```diff
@@ -42,23 +42,23 @@
 //! This library **does not** implement the complete protocol. There are several
 //! security-critical steps that must be handled by the calling application. We
 //! leave these undone intentionally, to allow increased flexibility in
-//! deploying the library, but this does mean that it requires cryptographic
-//! expertise to advise on security of the remaining components.
+//! deploying the library, but this does mean that a successful, correct
+//! deployment requires cryptographic expertise to advise on security of the
+//! remaining components.
+//!
 //! These caller requirements are highlighted throughout the documentation with
-//! the 🔒 symbol.
+//! the 🔒 symbol. At a high level, they include:
 //!
-//! 1. Networking. The protocol requires point-to-point channels between
-//! individual participants, as well as a UC-secure, synchronous broadcast
-//! mechanism. This library currently supports broadcast using the
-//! echo-broadcast protocol described by Goldwasser and Lindell[^echo], so the
-//! calling application only has to implement point-to-point channels.
-//! This may change in future library versions.
+//! 1. Networking. The protocol requires messages to be sent over secure
```

Comment:
I would just move up the details to this section. L52-53 will just confuse people.

Reply:
I rephrased these lines but I don't want this numbered list to have one massive bullet point. I like having this as the overview section with pointers to details. I moved the networking section up so it's immediately following, though.